samples/javascript-v4-azurefunction/src/functions/stateInputBinding.js

```diff
@@ -17,13 +17,16 @@ app.generic('StateInputBinding', {
         name: "req"
     }),
     extraInputs: [daprStateInput],
+    return: output.generic({
```

Comment:
Please import output, it is undefined right now.

Reply:
Thanks for the hint - I forgot to copy everything over from my working sample